src/components/pagination-button/PaginationButton.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { ShallowWrapper, shallow } from "enzyme";
+
+import { PaginationButton } from "./PaginationButton";
+import React from "react";
+
+describe("Given a pagination button", () => {
+    const pageValue = 1;
+    let jestFuntion: jest.Mock;
+    let wrapper: ShallowWrapper;
+
+    describe("that is not selected", () => {
+        beforeEach(() => {
+            jestFuntion = jest.fn();
+            wrapper = shallow(<PaginationButton onClick={jestFuntion} selected={false} value={pageValue} />);
+        });
+
+        describe("when the button is clicked", () => {
+            beforeEach(() => {
+                wrapper.find(".pagination-button").simulate("click");
+            });
+
+            it("calls the finctions passed as props", () => {

```

Comment:
Try and stick to Given, When, Then descriptions
```suggestion
            it("then it calls the function passed as props", () => {

```